src/main/java/frc/utils/applicationsutils/OnDeploy.java

```diff
@@ -0,0 +1,21 @@
+package frc.utils.applicationsutils;
+
+import javax.swing.*;
+
+public class OnDeploy {
+
+
+    public static void main(String[] args) {
+        startupLogic();
+    }
+
+    private static void startupLogic() {
+        //to be continued...
+    }
+
+    private static void testConnection() {
+        JFrame frame = new JFrame();
+        frame.setVisible(true);
+        frame.setDefaultCloseOperation(WindowConstants.EXIT_ON_CLOSE);
+    }
```

Comment:
What's this? you don't use it

Reply:
it's to test if the deploy program is working, it is purely for testing the OnDeploy class.